mrp_stock_info/i18n/mrp_stock_info.pot

```diff
@@ -0,0 +1,50 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* mrp_stock_info
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-02-09 14:31+0000\n"
+"PO-Revision-Date: 2015-02-10 13:43+0100\n"
+"Last-Translator: Oihane <oihanecruce@gmail.com>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+
+#. module: mrp_stock_info
+#: model:ir.model,name:mrp_stock_info.model_mrp_production_product_line
+msgid "Production Scheduled Product"
+msgstr ""
+
+#. module: mrp_stock_info
+#: field:mrp.production.product.line,product_qty_available:0
+msgid "Quantity On Hand"
+msgstr ""
+
+#. module: mrp_stock_info
+#: help:mrp.production.product.line,product_qty_available:0
+msgid ""
+"Current quantity of products.\n"
+"In a context with a single Stock Location, this includes goods stored at this Location, or any of its children.\n"
+"In a context with a single Warehouse, this includes goods stored in the Stock Location of this Warehouse, or any of its children.\n"
+"stored in the Stock Location of the Warehouse of this Shop, or any of its children.\n"
+"Otherwise, this includes goods stored in any Stock Location with 'internal' type."
+msgstr ""
+#. module: mrp_stock_info
+#: field:mrp.production.product.line,product_virtual_available:0
+msgid "Forecast Quantity"
+msgstr "a"
```

Comment:
Done in https://github.com/odoomrp/odoomrp-utils/commit/6bbaaba140ff6781de0c916c4ab7adff952ab87e